### PR TITLE
Backend hybrid annotations

### LINKED ISSUE #2981: Hybrid tracings: Change annotations endpoint so that skeleton and volume tracings can be returned
The format would change from

```
  content: {
    id: string,
    typ: string,
  },
  ...
```

to something like

```
  tracing: {
    skeleton: string,
    volume: string,
  },
  ...
```

As a first step, the endpoint would return only non-null for one of the properties, as we don't have hybrid tracings, yet.

### LINKED ISSUE #2982: Hybrid tracings: Allow explicit creation of hybrid tracing
When creating a new annotation, it should be possible to indicate (e.g., via an extra button) that a skeleton as well as a volume tracing should be created. We can use that route to further develop the feature for hybrid tracings and have it tested in a first phase. In the long run, it should be possible to convert all tracings to hybrids (e.g., by clicking a button or by just providing all hybrid functionality and lazily creating the skeleton or volume tracing).

### PR DESCRIPTION
**Pull request by fm3**
Backend-part only, to be merged into @philippotto ’s branch that does the corresponding frontend changes

- createExplorational can now also take `hybrid` as a type, next to `volume` and `skeleton`
- annotation json should now include ```
  tracing: {
    skeleton: string,
    volume: string,
  }
``` instead of  `content`

### Steps to test:
~~- create tracing / task / project~~
~~- download / upload should still work~~

### Issues:
- fixes #2981 
- fixes #2982 

------
- [ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- [x] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable
- [x] Ready for review

### CODE REVIEW

**Comment by philippotto** [comment]
I'm merging this PR into `fe-hybrid`, since it makes sense to adapt the front-end to the back-end changes there. @jstriebel Please review the back-end changes in #2985.

File: app/models/annotation/AnnotationMutations.scala [hunk 1]
```diff
@@ -62,15 +62,15 @@ class AnnotationMutations(val annotation: Annotation) extends BoxImplicits with
   def resetToBase(implicit ctx: DBAccessContext) = annotation.typ match {
     case AnnotationTypeSQL.Explorational =>
       Fox.failure("annotation.revert.skeletonOnly")
-    case AnnotationTypeSQL.Task if annotation.tracingType == TracingType.skeleton =>
+    case AnnotationTypeSQL.Task if annotation.skeletonTracingId.isDefined =>
```
Comment (jstriebel) [inline app/models/annotation/AnnotationMutations.scala]: let's stay with the `tracingType` check

Comment (jstriebel) [reply app/models/annotation/AnnotationMutations.scala]: maybe

File: tools/postgres/schema.sql [hunk 2]
```diff
@@ -42,8 +41,8 @@ CREATE TABLE webknossos.annotations(
   _task CHAR(24),
   _team CHAR(24) NOT NULL,
   _user CHAR(24) NOT NULL,
-  tracing_id CHAR(36) NOT NULL UNIQUE,
-  tracing_typ webknossos.ANNOTATION_TRACING_TYPE NOT NULL,
+  skeletonTracingId CHAR(36) UNIQUE,
+  volumeTracingId CHAR(36) UNIQUE,
```
Comment (jstriebel) [inline tools/postgres/schema.sql]: @fm3 Please add a comment here, that this _must_ be unique over the two columns, and is currently assured by the datastore, which generates those ids.

**Pull request merged**